arviz/data/converters.py

```diff
@@ -97,6 +99,10 @@ def convert_to_inference_data(obj, *, group="posterior", coords=None, dims=None,
     # Cases that convert to xarray
     if isinstance(obj, xr.Dataset):
         dataset = obj
+    elif isinstance(obj, xr.DataArray):
+        if obj.name is None:
+            obj.name = 'plot'
```

Comment:
If the name is None, we should set it to `x` to follow current convention with numpy arrays.

Reply:
Yes, true. I will change it.